etl_manager/meta.py

```diff
@@ -377,18 +418,22 @@ def update_column(self, column_name, **kwargs):
                     _validate_nullable(kwargs["nullable"])
                     c["nullable"] = kwargs["nullable"]
 
-                if "personal_data" in kwargs:
-                    _validate_personal_data(kwargs["personal_data"])
-                    c["personal_data"] = kwargs["personal_data"]
+                if "sensitivity" in kwargs:
+                    _validate_column_sensitivity(kwargs["sensitivity"])
+                    self._check_valid_column_sensitivity(kwargs["sensitivity"])
+                    c["sensitivity"] = kwargs["sensitivity"]
 
-                if "special_category_data" in kwargs:
-                    _validate_special_category_data(kwargs["special_category_data"])
-                    c["special_category_data"] = kwargs["special_category_data"]
+                if "redacted" in kwargs:
+                    _validate_redacted(kwargs["redacted"])
+                    c["redacted"] = kwargs["redacted"]
 
             new_cols.append(c)
 
         self.columns = new_cols
 
+        # Update table sensitivity
+        self.sensitivity = []
```

Comment:
If agree with not setting sensitivity (first comment) then maybe change to something like `self.update_sensitivity` where you calculate it and validate the sensitivity output.